### PR TITLE
Story demonstrating transform using SceneView

### PR DESCRIPTION
**Pull request by BrennaCourtney**
### Summary of changes 🔍 
Part 2 of integrating transforms exploration: this PR creates a story demonstrating the "Viewer" side of transforms: that is, switching transforms on and off.

image

### Testing 🧪
https://601c6b2fcd385c002100f14c-errnbfozut.chromatic.com/

Head to Componenets -> SceneView -> Gizmo. Use the checkboxes to toggle the transforms of two tanks in the model. They should operate independently.

This story simulates the Viewer receiving transformedElementItems in response to changing property values -- at times some transforms may be turned on while others are turned off, which is specified by elements being passed in transformedElementItems.

### Checklist ✔️
- [ ] Linked associated issue (if present)
- [ ] Added relevant labels
- [ ] Requested developer reviews
- [ ] Request UI review from design / PM
- [ ] Verify all code checks are passing

### CODE REVIEW

File: src/Components/3DV/SceneView.tsx [hunk 1]
```diff
@@ -1680,6 +1683,132 @@ function SceneView(props: ISceneViewProps, ref) {
...
+                        // get mesh id for the previously transformed element
+                        const prevTransParentMeshId =
+                            previouslyTransformedElement.meshId;
+                        // if the parentMeshIds to be transformed DOES NOT include the previouslyTransformedParentMeshId,
+                        // then that means the transformation no longer applies and the element should be reset
+                        if (!tEIParentMeshIds.includes(prevTransParentMeshId)) {
+                            const prevTransParentMesh: BABYLON.Mesh =
+                                meshMap.current?.[prevTransParentMeshId];
+                            // setting rotation back to original
+                            if (prevTransParentMesh) {
```
Comment (msnyder-msft) [inline src/Components/3DV/SceneView.tsx:1720]: could we pull some of these things out to the `SceneView.Utils` file? Like take in the 2 objects but do all the mapping of properties in the uility to simplify this file?

File: src/Components/3DV/SceneView.tsx [hunk 2]
```diff
@@ -1680,6 +1683,132 @@ function SceneView(props: ISceneViewProps, ref) {
...
+
+        const pTParentMeshIds: string[] = previouslyTransformedElements.current.map(
+            (pTE) => pTE.meshId
+        );
+
+        const transform = transformedElementItem.transform;
+
+        // only add parentMesh to previouslyTransformedElements ONCE for the ORIGINAL status
+        if (!pTParentMeshIds.includes(transformedElementItem.parentMeshId)) {
+            previouslyTransformedElements.current.push({
```
Comment (msnyder-msft) [inline src/Components/3DV/SceneView.tsx:1786]: could pull this creation logic to the util as well. Pass in a trasnform/mesh and return back the object?

**Review by msnyder-msft — APPROVED** [review]

File: src/Components/3DV/SceneView.tsx (hunk 1 above)
Comment (BrennaCourtney) [reply src/Components/3DV/SceneView.tsx:1721]: done! (also used them to refactor gizmo functions)

File: src/Components/3DV/SceneView.tsx (hunk 2 above)
Comment (BrennaCourtney) [reply src/Components/3DV/SceneView.tsx:1778]: done! (also used them to refactor gizmo functions)

**Pull request merged**